### ISSUE TITLE
Some layers don't render in exported HTML file

### ISSUE THREAD

**Issue by jmcdermottsweeney** [airbnb/lottie-web#1558]
I've been searching the web and been through the issue page but nothing seems to address my issue, even though others seem to have similar problems. I have a composition ("text") that has shape layers (converted from text) that use the Stroke effect and Trim Paths. When I use Bodymovin to export the comp, only a couple of the characters in the text composition render in the browser (and since some of the characters are the same, those layers are just duplicates). I'm really confused as to why Bodymovin is rendering some of these characters perfectly and can't seem to find a solution anywhere.

Note: There is a comp in there (titled "burst") that uses effects that I now realize Bodymovin does not support – but when I remove it from the text comp, I still run into the same issue as stated above. Not really concerned about getting that comp in there.

Here are my specs:
Working in After Effects CC 2019
Bodymovin v5.4.4

[balance-counter.aep.zip](https://github.com/airbnb/lottie-web/files/3043827/balance-counter.aep.zip)

**Comment by jmcdermottsweeney** [comment]
UPDATE: Was inspecting the animation in the browser (I'm using Safari, but I've had the same results with Chrome). I found these <clippath> tags:

<img width="1280" alt="Screen Shot 2019-04-04 at 10 26 57 AM" src="https://user-images.githubusercontent.com/41841558/55563646-528c0e80-56c4-11e9-8316-25b42bc597a8.png">

When I deleted them, the composition rendered as I wanted!

<img width="1280" alt="Screen Shot 2019-04-04 at 10 27 15 AM" src="https://user-images.githubusercontent.com/41841558/55563757-849d7080-56c4-11e9-9ffb-82bb913ed2ce.png">

Hopefully this helps with troubleshooting. Would love to know how to solve this in After Effects.
